strings/src/main/res/values/strings.xml

```diff
@@ -705,4 +705,7 @@
     <string name="security_error">Collect can\'t connect securely to the server at %s. Did you enter the URL correctly?</string>
     <string name="failure">Failure</string>
     <string name="form_update_succeeded">Form update succeeded</string>
+    <string name="in_app_recording">Use in-app audio recording</string>
```

Comment:
What do you think about flipping this and going with "Use external app for audio recording" (true by default)? That way it most likely won't need to be translated again. We can absolutely change the wording if we find we need to but at least we send something to translators that's likely to stick around.

Reply:
Good idea!